### ISSUE TITLE
Phase Number Rework.

### ISSUE THREAD

**Issue by IndigoAmethyst** [nonamecrackers2/crackers-wither-storm-mod#1188]
An existent issue is that the Phase numbers do not make sense, I have noticed that many people have become confused over this. Here are the issues:

- The first phase is called "Phase 0", which is not something many may realise when finding out about the phases.
- The Phase after Phase 5 is.. 5.5, which doesn't make any sense and I constantly see people not even knowing this exists. Also, this cannot accessed via /witherstormmod phase command which pretty much stops most creative users from discovering it. Same issue with Phase 7.5.

How I would recommend to fix:

Just make the 10 Phases actually go from 1-10 without any random decimals between.

Please consider my suggestion.

**Comment by IndigoAmethyst** [comment]
Another issue related to this is #1187

**Comment by myron-obsolete-account** [comment]
to me phase 0 for the "first" phase makes sense because phase 1 is the first phase where the storm actually starts evolving. phase 0 is not really anything wither storm at all, but just the normal wither with a command block.

**Comment by nonamecrackers2** [comment]
In programming, 0 is considered the first element of arrays as an example. I just went along with it, and changing this now would make no sense and would just confuse the players already used to it. 

Those sub phase numbers are really just meant to help identify the differences in the main phases, they have no correlation in code.

Making the sub phases their own phases (which is what I assume you’re suggesting I do) would require extensive code rewriting (primarily due to the hard-coded nature of the phase system, which is a fault of mine when I started the mod) and would not be feasible. I could possibly make the phase command accept decimals and it would adjust the consumed entities and phase accordingly to apply the sub phase (also sub phases are controlled by the consumed entity counter, not the phase counter which should be mentioned), however you’ll have to open a new suggestion issue to request that.
